src/Chapter1.hs

```diff
@@ -619,8 +624,11 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = lastDigit + secLastDigit
+  where
+    lastDigit = mod n 10
```

Comment:
You can notice that this is the same as `lastDigit` function from the previous task 🙂 

Also, here, you also need to consider the case with negative numbers 😉